rightSideView.py

```diff
@@ -0,0 +1,34 @@
+# Definition for a binary tree node.
+# class TreeNode(object):
+#     def __init__(self, x):
+#         self.val = x
+#         self.left = None
+#         self.right = None
+
+#Accepted by leetcode
+#Time complexity - O(n) because we are travesing through each node
+#Space complexity - O(n) we are using extra space queue, result
+
+
+class Solution(object):
+    def rightSideView(self, root):
+        """
+        :type root: TreeNode
+        :rtype: List[int]
+        """        
+        if root == None:
+            return []        
+        queue = []
+        result = []
+        queue.append(root)
+        while len(queue) > 0:
+            result.append(queue[-1].val)

```

Comment:
what's the time complexity to get an element at index i from a list in python?

Reply:
Time complexity to get an element at any index in a list is O(1) since we know the index, we can directly go there and access it.